build/crds-gen.sh

```diff
@@ -20,7 +20,7 @@ installOperatorSDK() {
     rm -rf "${OPERATOR_SDK_BUNDLE}"
     exit 1
   fi
-  chmod +x operator-sdk-"${OPERATOR_SDK_VERSION}"-x86_64-linux-gnu && sudo mkdir -p /usr/local/bin/ && sudo cp "${OPERATOR_SDK_BUNDLE}" /usr/local/bin/operator-sdk && rm "${OPERATOR_SDK_BUNDLE}"
```

Comment:
Could you please add `arch` command call instead of hardcoded `x86_64` architecture to conform all possible Operator SDK OS installations